src/zap/zapper.cpp

```diff
@@ -319,6 +318,18 @@ ZapperOptions::~ZapperOptions()
         delete [] m_repositoryDir;
 }
 
+void ZapperOptions::SetZapRequiredCompilerFlags(void)
+{
+    m_compilerFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_RELOC);
+    m_compilerFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_PREJIT);
+
+#if defined(_TARGET_ARM_)
+# if defined(PLATFORM_UNIX)
+    m_compilerFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_RELATIVE_CODE_RELOCS);
+# endif // defined(PLATFORM_UNIX)
```

Comment:
Looks fine to me. (I would call the method just `ZapperOptions::SetCompilerFlags`.)